backend/Money.Api/Controllers/FileController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Money.Api.Dto.Files;
+using Money.Business;
+using Money.Business.Services;
+using OpenIddict.Validation.AspNetCore;
+
+namespace Money.Api.Controllers;
+
+[ApiController]
+[Authorize(AuthenticationSchemes = OpenIddictValidationAspNetCoreDefaults.AuthenticationScheme)]
+[Route("[controller]")]
+public class FileController(FileService fileService) : ControllerBase
+{
+    [HttpPost]
+    public async Task<FileDto> UploadFile(IFormFile file)
+    {
+        var uploadedFile = await fileService.Upload(file);
```

Comment:
Argument type 'Microsoft. AspNetCore. Http. IFormFile' is not assignable to parameter type 'IFormFile'.